workorders.lic

```diff
@@ -872,7 +893,6 @@ class WorkOrders
           fput("drop my #{left_hand}")
         end
         fput('get logbook') unless [left_hand, right_hand].grep(/logbook/i).any?
-      end
```

Comment:
Should this end be removed?
Looks like it will cause issues.

Reply:
Looks like it was removed by mistake. Correcting. Good catch.